htmresearch/frameworks/nlp/classify_network_api.py

```diff
@@ -57,6 +57,13 @@ def __init__(self,
       raise RuntimeError("Must specify a network config dict")
 
 
+  def getEncoder(self):
+    """
+    Returns the encoder instance for the model.
+    """
+    return self.encoder
```

Comment:
This is not going to work properly after saving and loading. It should instead do:
```
return self.sensorRegion.getSelf().encoder
```
You can then get rid of self.encoder.